src/components/ProjectsPage/Projects.js

```diff
@@ -68,18 +63,100 @@ const Projects = ({
   const [selectedProjectsState, setSelectedProjectsState] = useState('active')
   const [sortProjectId, setSortProjectId] = useState('byName')
   const [deletingProjects, setDeletingProjects] = useState({})
+  const [jobsFilter, setJobsFilter] = useState({
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    }
+  })
+  const [workflowsFilter, setWorkflowsFilter] = useState({
+    groupBy: GROUP_BY_WORKFLOW,
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    },
```

Comment:
why do you need to have value property? You can set the array directly to dates property

Reply:
@mavdryk the reason is because of filters logic we have

example:
if (filters.dates.value[0]) {
      params.start_time_from = filters.dates.value[0].toISOString()
    }